www/layouts/home.html

```diff
@@ -98,7 +98,7 @@ <h2>Next Generation OpenCourseWare</h2>
       <div class="d-flex flex-column justify-content-center">
         <h2>GIVE NOW</h2>
         <h2>Your Donation Makes a Difference</h2>
```

Comment:
Are we allowed to have two `<h2>` tags?

Reply:
It looks like they only discourage multiple header-element tags for `h1`: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements

I think we are all over the place with our header element tags so it may be best to just convert them all to span or div tags with the equivalent styling so the design doesn't change. I can do that in a future PR